android/src/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -268,10 +268,14 @@ public float getBottomButtonsHeight()
 
   public void showMapButtons(boolean show)
   {
-    if (show)
+    if (show) {
       UiUtils.show(mFrame);
-    else
+      showButton(true, MapButtons.zoom);
+    }
+    else {
       UiUtils.hide(mFrame);
+      showButton(false, MapButtons.zoom);
```

Comment:
The zoom buttons are inside the frame, so there is no need to hide them if we hide the parent frame.

Reply:
if that line is not added then the zoom buttons pop up again when we enter the edit mode.

Reply:
Why do zoom buttons pop up again if this line is not added if the frame is hidden? Is the frame displayed again for some reason in the edit mode? What causes it? How can we fix the root cause?

Reply:
This PR might be a temporary solution. But I will certainly try to find the root cause.

Reply:
Temporary solutions easily stack on each other and then lead to other unexpected issues. Let's focus on fixing the root cause.

Reply:
So shall I close this pull request then?

Reply:
It would be great if you find a root cause, and fix it in this PR, without creating another one.

Reply:
The zoom buttons are included in the mainframe and disappear and reappear as other buttons present there.
The thing is zoom buttons have a separate setting of disabling/enabling them unlike the other mainframe buttons, this is causing them to act differently and needs to add those statements there.
I'll try to find a different approach if this doesnt sound right to you.